src/views/template/history.html

```diff
@@ -0,0 +1,6 @@
+<div class="panel panel-default">
+    <div class="panel-heading">
+        History
```

Comment:
Wrap text in `{{#l}}History{{/l}}`, and add `History` to the `nl.json`, with a translation. Same thing goes for the other non-translated strings/variables.